.github/workflows/build.yaml

```diff
@@ -56,6 +56,54 @@ jobs:
       - name: Check for no generated changes
         run: git update-index --refresh && git diff-index HEAD
 
+      - name: Package SDK for smoke tests
+        run: |
+          cd packages/sdk
+          pnpm pack --out linear-sdk.tar.gz
+
+      - name: Upload SDK artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: linear-sdk
+          path: packages/sdk/linear-sdk.tar.gz
+          retention-days: 1
+
+  smoke:
+    name: Smoke Test ${{ matrix.module }} (Node ${{ matrix.node }} / TS ${{ matrix.typescript }})
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ["20", "22", "24"]
+        typescript: ["5.7", "5.8", "5.9"]
```

Comment:
Maybe we just do `node: ["20", "24"]` and `typescript: ["5.7", "5.9"]`? Seems we essentially cover the same bases implicitly, with 8 CI runs rather than 18.